Source/Assets/PlayFabEditorExtensions/Editor/Scripts/PlayFabEditorSDK/PlayFabEditorHttp.cs

```diff
@@ -5,16 +5,23 @@
 using System.IO;
 using PlayFab.PfEditor.Json;
 using PlayFab.PfEditor.EditorModels;
+using UnityEngine.Networking;
 
 namespace PlayFab.PfEditor
 {
     public class PlayFabEditorHttp : UnityEditor.Editor
     {
         internal static void MakeDownloadCall(string url, Action<string> resultCallback)
         {
+#if UNITY_2018_2_OR_NEWER
+            UnityWebRequest www = UnityWebRequest.Get(url);
+            PlayFabEditor.RaiseStateUpdate(PlayFabEditor.EdExStates.OnHttpReq, url, PlayFabEditorHelper.MSG_SPIN_BLOCK);
+            EditorCoroutine.Start(PostDownload(www, (response) => { WriteResultFile(url, resultCallback, response); }, PlayFabEditorHelper.SharedErrorCallback), www);
+#elif UNITY_5_3_OR_NEWER
```

Comment:
Since we're deprecating everything before 5.6, I don't think we need to specify this.
You can just #else